src/components/ArtObjectPageComponents/PanelDetails/index.jsx

```diff
@@ -195,30 +200,30 @@ class Image extends Component {
 
     // This indicates that there was an error with rendering the Zoom component
     const { didCatchFailure } = this.state;
-    const showZoomImageView = Boolean(!didCatchFailure && object.id && activeImageIndex === 0);
     const renditionsExist = renditions?.length > 0;
+    const showZoomImageView = Boolean(!didCatchFailure && object.id && !renditionsExist && activeImageIndex === 0);
```

Comment:
We will only show the Zoom image view if renditions are not available as the UX in that way will be kind of clunky.